app/_hub/kong-inc/response-transformer-advanced/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 name: Response Transformer Advanced
 publisher: Kong Inc.
-version: 0.36-x
+version: 1.3-x
```

Comment:
@levfishbluefish , do we need to create a 0.36-x version for this page? I don't know the process for backdating Enterprise plugins, especially given that there are no changes.

Reply:
Going to assume not, since I checked many other files as examples and see the same pattern.

Reply:
sorry yeah this was a test change and I didn't push up reverting it!